.pre-commit-hooks.yaml

```diff
@@ -28,3 +28,16 @@
   pass_filenames: false
   additional_dependencies: []
   minimum_pre_commit_version: "2.9.2"
+- id: uv-sync
+  name: uv-sync
+  description: "Automatically run 'uv sync' on your workspace after a checkout or pull"
+  entry: uv-sync-hook
+  args:
+    - "--locked"
+  language: python
+  always_run: true
+  pass_filenames: false
+  minimum_pre_commit_version: "2.9.2"
+  stages:
```

Comment:
Should we add `post-rewrite`? Lots of folks use a .gitconfig which uses rebase on pull instead of a merge.